templates/email_response.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block title %}
+Email Confirmation page
+{% endblock %}
+
+{% block head %}
+    <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
+{% endblock %}
+
+{% block body %}
+<div class="flex items-center justify-center min-h-screen p-5 bg-white min-w-screen">
+            <div class="max-w-xl p-8 text-center text-gray-800 bg-gray-200 shadow-xl lg:max-w-3xl rounded-3xl lg:p-12">
+                <h3 class="text-2xl color-white">Your email has been confirmed!</h3><br>
+                <div class="flex justify-center">
+                    <img src="{% static 'img/svg_temp.png' %}" width="100" alt="Your Email has been verified!">
+                </div><br>
+                <p>Now you can login to your account!</p><br>
+            <a href="https://roboticsclub.mnnit.ac.in/user/login/" class=" my-1.5 px-1.5 py-1.5 text-white bg-red-500 rounded">Back to Login page</a>
```

Comment:
@Lalit1471 use django url pattern instead of hardcoding url in href

Reply:
Updated